tools/extractReleaseNotes.js

```diff
@@ -0,0 +1,48 @@
+// Parse PR descriptions for release notes
+const fetch = require("node-fetch");
+const fs = require("fs");
+
+// Set this path to the cached json file to avoid hitting the github api limit while developing
+const CACHED_RESPONSE_PATH = "";
+// Adjust this variable to the last PR that shouldn't be included in the release notes
+const LAST_PR = 0;
+
+const RELEASE_NOTES_REGEX = /### Mailable description[^:]*:\r\n([^#]*)/;
```

Comment:
I've made the regex a little more robust, so it won't break if the mailable description contains a # or there is `\n` instead of `\r\n`, thanks for the suggestions :)